src/main/java/frc/robot/Constants.java

```diff
@@ -129,7 +162,7 @@ public static final class Mod3 { //TODO: DONE
         }
     }
 
-    public static final class AutoConstants { //TODO: The below constants are used in the example auto, and must be tuned to specific robot
+    public static final class AutoConstants { //TODO: DONE
```

Comment:
General comment: Remove TODOs if they are done. I personally search for TODO to see if there's any straggling items that weren't done in a review, so this may confuse others.